docs/topics/mta-web-assessment-module.adoc

```diff
@@ -0,0 +1,111 @@
+
+// Get updated image for MTA
+image::mta-assessment-apps-01.png[MTA application inventory]
+
+. Open the {ProductName} user interface and click *Application Inventory* in the left menu bar. *Application assessment* appears in the main pane, listing all the available questionnaires. 
+. Click *Create new*. A *New application* dialog appears.
+
+// Get updated image for MTA
+image::mta-assessment-new-app-01.png[MTA application inventory]
+
+[start=3]
+. In the form, enter the following information:
+.. *Name* of the new application (mandatory)
+.. *Description* of the application (optional)
+.. *Business service* is thepurpose of the application (optional)
+.. *Tags* software tags that charecterize the application (mandatory)
+.. *Owner* select regitered software owner from drop-down list (optional)
+.. *Contributors* select contributors from drop-down list (optional)
+.. *Comments* enter comments on the migrated application
+. Click *Source code* and enter the following fields.
+.. *Repository type* select *Git* or *Subversion* (mandatory)
+.. *Source Repository* paste the URL of the repository where the software code is saved (mandatory)
+.. *Branch* enter the repository branch that is to be migrated (mandatory)
+.. *Root path* [get info] (optional/mandatory)
+. Click *Binary* and enter the following fields.
+.. *Group* [get info] (optional/mandatory)
+.. *Artifact* [get info] (optional/mandatory)
+.. *Version* software version of the application
+.. *Packaging* [get info] (optional/mandatory)
+. Click *Create*.
+
+.To open a saved application
+
+// Get updated image for MTA
+image::mta-assessment-apps-01.png[MTA aplication inventory]
+
+. Open the {ProductName} user interface and click *Application Inventory* in the left menu bar. *Application assessment* appears in the main pane, listing all the available questionnaires. 
+. Review all the applications and click *Take* to the right of the selected questionnaire. The main pane shows a list of defined assessments.
+
+// Get updated image for MTA
+image::assessment-view-assessment-02.png[]
+
+[start=3]
+. Right click one of the defined assessments and select *view assessment*. The main pane shows the list of questions in the assessment, separated into categories.
+
+// Get updated image for MTA
+// this image name is incorrect
+//image::assessment-view-assessment-03.png[]
+image::assessment-view-assessment-view-03.png
+
+// Get updated image for MTA
+image::mta-assessment-app-questions-01.png[Application assessment questions]
+
+[start=4]
+. On the left of the main pane a list of question categories, and in the main pane are a list of questions for the category appears with a drop-down list of answers for each question.
+.. Click the down arrow to the right of one of the questions. A list of answer options appears below the list of questions.
+.. Select one of the answers under *Answer choice*.
+. Click *Next* to continue to *Application details*.
+.. Click the down arrow
```

Comment:
Tell them to answer the questions. It's obvious, but you don't want to just say Click *Next* without telling them what to do.  Are all questions mandatory? 
Van he only click *Save as draft* when they finish? This is a long questionnaire; tell them earlyon they can save a draft and come back to it.